WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -98,6 +107,9 @@ private extension TotalsView {
                     .redacted(reason: redacted ? [.placeholder] : [])
             }
             .foregroundColor(Color.primaryText)
+            .transition(.opacity)
```

Comment:
I don't think this has any effect. If it does, should it also be present on the subtotal version?

```suggestion
```

Reply:
Good observation. Yes, it has a small effect to fade from shimmering to normal text views. However, it should also apply to the subtotal version.

Transition has to be applied together with animation that is triggered when shimmering value changes `.animation(.default, value: totalsViewModel.isShimmering)`. I will move them to a single place to apply for all `totalsFieldsView`:

```
                    totalsFieldsView
                        .transition(.opacity)
                        .animation(.default, value: totalsViewModel.isShimmering)
 ```

### Without transition + animation

https://github.com/user-attachments/assets/59ab2009-939f-48b1-bed8-237bb338ec62

### With transition + animation

https://github.com/user-attachments/assets/0dd73198-5adc-4618-a1ac-7be977a86d4d

Reply:
Fixed / improved here https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/d0589ac41966564b4f2ee028717e53afa0e1543e